projects/quml-library/src/lib/player/player.component.ts

```diff
@@ -93,8 +94,9 @@ export class PlayerComponent implements OnInit, AfterViewInit {
     this.showIndicator = false;
     this.noWrapSlides = true;
     this.questions = this.QumlPlayerConfig.data.children;
-    this.timeLimit = this.QumlPlayerConfig.data.timeLimit ?
-      this.QumlPlayerConfig.data.timeLimit : (this.questions.length * 350000);
+    this.timeLimit = this.QumlPlayerConfig.data.timeLimits.totalTime ?
+      this.QumlPlayerConfig.data.timeLimits.totalTime : (this.questions.length * 350000);
```

Comment:
This logic should have been changed to count up and count down already by diptesh, don't override it again